src/numerical_irreducible_decomposition.jl

```diff
@@ -4,7 +4,7 @@ export NumericalIrreducibleDecomposition,
     regeneration,
     witness_sets,
     decompose,
-    n_components,
+    ncomponents,
```

Comment:
@PBrdng You should still keep for now an alias `n_components = ncomponents` around and export it. Otherwise this could break code